app/lib/pulp/v3/api.rb

```diff
@@ -1,11 +1,12 @@
 require 'pulpcore_client'
```

Comment:
This file will need to be split out in order to work with the new bindings.  Multiple API endpoints here will have methods with the same name.